.github/workflows/ci.yml

```diff
@@ -219,175 +178,51 @@ jobs:
-            || github.ref == 'refs/heads/master'
-        name: "[run-api-ci] PHP 8.3 / MariaDB / Robot / API"
-        runs-on: ubuntu-latest
-        env:
-            PROGRESS_TYPE: plain
-            SPRYKER_PLATFORM_IMAGE: spryker/php:8.3
-            TRAVIS: 1
-            ROBOT_TESTS_ARTIFACTS_BUCKET_REGION: eu-west-1
-            SPRYKER_CURRENT_REGION: EU
-            SPRYKER_DYNAMIC_STORE_MODE: true
-        steps:
-            -   uses: actions/checkout@v4
+                    docker/sdk cli vendor/bin/console queue:setup
+                    docker/sdk cli vendor/bin/console setup:init-db
+                    docker/sdk cli vendor/bin/console data:import --config=data/import/common/minimal.yml
+                    docker/sdk cli vendor/bin/console search:setup:sources
+                    docker/sdk cli vendor/bin/glue glue-api:controller:cache:warm-up
+                    docker/sdk cli vendor/bin/glue api:generate:documentation
 
-            -   name: Install packages
-                run: |
-                    sudo apt-get update
-                    sudo apt-get install -y python3-pip
-                    pip3 install --upgrade pip
-                    pip3 install awscli
-            -   name: Install robotframework-suite-tests folder
+            -   name: Publish & Sync
                 run: |
-                    cd ./data && composer require "spryker/robotframework-suite-tests:dev-master" --dev --no-interaction
-                    cp -r vendor ../vendor
-            -   name: Install Project
-                continue-on-error: false
-                run: |
-                    git clone https://github.com/spryker/docker-sdk.git ./docker
-                    docker/sdk boot deploy.ci.acceptance.mariadb.robot.yml
-                    sudo bash -c "echo '127.0.0.1 backend-api.eu.spryker.local backend-api.us.spryker.local backend-gateway.eu.spryker.local backend-gateway.us.spryker.local backoffice.eu.spryker.local backoffice.us.spryker.local glue-backend.eu.spryker.local glue-backend.us.spryker.local glue-storefront.eu.spryker.local glue-storefront.us.spryker.local glue.eu.spryker.local glue.us.spryker.local mail.spryker.local mp.eu.spryker.local mp.us.spryker.local queue.spryker.local scheduler.spryker.local spryker.local swagger.spryker.local yves.eu.spryker.local yves.us.spryker.local' >> /etc/hosts"
-                    docker/sdk up -t
-            -   name: Custom commands
-                continue-on-error: false
+                    docker/sdk testing console queue:worker:start --stop-when-empty
+
+            -   name: Fixtures
                 run: |
-                    docker/sdk cli composer dump-autoload -o -a --apcu
-                    SPRYKER_CURRENT_REGION=EU docker/sdk console queue:worker:start --stop-when-empty
-            -   name: Run Tests
+                    docker/sdk testing codecept fixtures
+
+            -   name: Glue Acceptance CheckoutMinimumWorkflow
```

Comment:
According to the acceptance criteria, we also need to have the yves checkout workflow tested besides of the glue checkout workflow